quick_setup_samples/quck_setup_for_macos_x64.md

```diff
@@ -0,0 +1,68 @@
+# Quick Setup for macos x64
+
+In case you would like to try a fool-proof setup for macos from scratch, you can try the following steps.
+You can also skip some of the steps if it is not applicable, for example, you don't need to install Java 1.8 if you have it already installed.
+
+You can also try your own variations. But the example provided here makes it easy for anybody to setup a powpeg node from scratch in an easy way.
+
+## Steps
+
+1. Install Java OpenJDK 1.8 from the binaries (if it is not installed yet)
+    1. You can download it from here: `https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u-2021-05-08-08-14/OpenJDK8U-jdk_x64_mac_hotspot_2021-05-08-08-14.tar.gz`
+        1. `cd` to the directory where you want to download it. It could be like `cd /Library/Java/JavaVirtualMachines/`
+        2. Download it like this: `sudo curl -o adoptopenjdk-8.tar.gz -L https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u-2021-05-08-08-14/OpenJDK8U-jdk_x64_mac_hotspot_2021-05-08-08-14.tar.gz`
+        3. Unzip it like this: `sudo tar -zxvf adoptopenjdk-8.tar.gz`
+        4. The unzipped directory could be named like this `jdk8u302-b01`. Run an `ls` command to see it.
+    2. Add it to the `.bash_profile`.
+        1. Open the `.bash_profile` file with `nano` like this `nano ~/.bash_profile` and add the following 2 `export` lines:
+        2. `export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk8u302-b01/Contents/Home/bin`
+        3. `export PATH="$JAVA_HOME/bin:$PATH"`
+        4. Save it and run the command: `source ~/.bash_profile` and restart the terminal. After this, the `java -version` command should return info.
+2. Install bitcoind
+    1. `cd` to the directory where you want to download it. It could be `/Library/Bitcoin` (if it doesn't exit, you can create it running the command `sudo mkdir /Library/Bitcoin`)
+    2. You can download version 0.18.1 (ideal for regtest) from here: `https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-osx64.tar.gz`
+    3. You can download version 24.0.1 `https://bitcoincore.org/bin/bitcoin-core-24.0.1/bitcoin-24.0.1-x86_64-apple-darwin.tar.gz`
```

Comment:
Isn't this a secondary option when the user is not interested in the `0.18.1` version? If so, why numbered it as the second step when the rest of the steps refer to the `0.18.1` version?